syncany-lib/src/main/java/org/syncany/operations/daemon/DaemonOperation.java

```diff
@@ -70,82 +73,96 @@
  * @author Philipp C. Heckel <philipp.heckel@gmail.com>
  * @author Pim Otte 
  */
-public class DaemonOperation extends Operation {	
-	private static final Logger logger = Logger.getLogger(DaemonOperation.class.getSimpleName());	
+public class DaemonOperation extends Operation {
+	private static final Logger logger = Logger.getLogger(DaemonOperation.class.getSimpleName());
 	public static final String PID_FILE = "daemon.pid";
 
 	private DaemonOperationOptions options;
 	private File pidFile;
-	
+
 	private WebServer webServer;
 	private WatchServer watchServer;
 	private ControlServer controlServer;
 	private LocalEventBus eventBus;
 	private DaemonConfigTO daemonConfig;
-	private PortTO portTO;	
+	private PortTO portTO;
 
 	public DaemonOperation() {
 		this(new DaemonOperationOptions(DaemonAction.RUN));
 	}
-	
+
 	public DaemonOperation(DaemonOperationOptions options) {
-		super(null);	
-		
+		super(null);
+
 		this.options = options;
-		this.pidFile = new File(UserConfig.getUserConfigDir(), PID_FILE);		
+		this.pidFile = new File(UserConfig.getUserConfigDir(), PID_FILE);
 	}
 
 	@Override
-	public DaemonOperationResult execute() throws Exception {		
+	public DaemonOperationResult execute() throws OperationException {
 		logger.log(Level.INFO, "Starting daemon operation with action " + options.getAction() + " ...");
-		
+
 		switch (options.getAction()) {
 		case LIST:
 			return executeList();
 
 		case ADD:
-			return executeAdd();
-
+			try {
```

Comment:
The try/catch blocks in the switch/case look odd. Can we maybe just throw OperationException in the execute*() methods?